docs/fos_http_cache.md

```diff
@@ -1,21 +1,30 @@
 # FOSHttpCacheBundle usage in eZ
 
-Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
-features:
+_This doc explains how this bundle integrates with FOSHttpCacheBundle internally, and is aimed at core developers._
 
-* Tags purging/baning (since version 1.8)
-* Http cache purge/ban
-* User context hash
+
+eZ Platform HttpCache _(this bundle)_ uses [FOSHttpCacheBundle 1.x][fos] in the following way:
+- When `local` purge type is set, system uses a custom Symfony Proxy tag aware data store, and hence does not
+  use FOSHttpCacheBundle directly. However FOSHttpCache(Bundle) can still be used to manipulate responses, incl tagging.
+- When `http` purge type is set, system uses FOSHttpCache configured with varnish as proxy client.
+
+As HttpCache in eZ Publish/Platform pre dates FOSHttpCache, we have own abstractions for:
```

Comment:
`predates` (one word)

Reply:
I think André meant "existed before". "predate" is like in "predator" is it not ? :)

Reply:
I did, but is it predator? :) :P